tailscale/config.yaml

```diff
@@ -27,4 +27,5 @@ map:
   - share:rw
 schema:
   tags: ["match(^tag:[a-zA-Z0-9]-?[a-zA-Z0-9]+$)?"]
+  proxy_port: match(^(?:[1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$)?
```

Comment:
We should not add-on configuration for this; instead, assign a port for it. A user can change such ports from the UI.